mods/ra/rules/structures.yaml

```diff
@@ -1352,7 +1352,6 @@ AFLD:
 		LongDesc: Reveals an area of the map\nand cloaked enemy units.
 		SelectTargetSpeechNotification: SelectTarget
 		EndChargeSpeechNotification: SpyPlaneReady
-		CameraActor: camera.spyplane
```

Comment:
The simplest "fix" would be to change this to a paradrop power and drop a single camera.spyplane.